src/Controller/ListPageRssController.php

```diff
@@ -0,0 +1,188 @@
+    return new static(
+      $container->get('entity_type.manager'),
+      $container->get('event_dispatcher'),
+      $container->get('theme.manager')
+    );
+  }
+
+  /**
+   * Render the RSS feed of a list page result.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   The list page node.
+   *
+   * @return string
+   *   The rendered RSS feed.
+   */
+  public function build(NodeInterface $node): Response {
+    // Build the render array for the RSS feed.
+    $default_title = $node->getTitle() . ' - RSS';
+    $default_link = $node->toUrl('canonical', ['absolute' => TRUE])->toString(TRUE);
+    $build = [
+      '#theme' => 'oe_list_pages_rss',
+      '#title' => $default_title,
+      '#link' => $default_link->getGeneratedUrl(),
+      '#description' => '',
+      '#language' => $node->language()->getId(),
+      '#copyright' => $this->getCopyright(),
+      '#image_url' => $this->getDefaultImageUrl(),
+      '#image_title' => $default_title,
+      '#image_link' => $default_link->getGeneratedUrl(),
+      '#channel_elements' => [],
+      '#items' => $this->getItemList($node),
+    ];
+
+    $cache_metadata = CacheableMetadata::createFromRenderArray($build);
+    $cache_metadata->addCacheableDependency($node);
+    $cache_metadata->addCacheableDependency($default_link);
+    $cache_metadata->applyTo($build);
+
+    // Dispatch event to allow modules to alter the build before being rendered.
+    $event = new ListPageRssBuildAlterEvent($build, $node);
+    $this->eventDispatcher->dispatch(ListPageEvents::ALTER_RSS_BUILD, $event);
+    $build = $event->getBuild();
+
+    // Create the response and add the xml type header.
+    $response = new CacheableResponse('', 200);
+    $response->headers->set('Content-Type', 'application/rss+xml; charset=utf-8');
+
+    // Render the list and add it to the response.
+    /** @var \Drupal\Core\Render\RendererInterface $renderer */
+    $renderer = \Drupal::service('renderer');
+    $output = (string) $renderer->renderRoot($build);
+    if (empty($output)) {
+      throw new NotFoundHttpException();
+    }
+    $response->setContent($output);
+    $response->addCacheableDependency($cache_metadata);
+
+    return $response;
+  }
+
+  /**
+   * Returns a list of items to be rendered.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   The node containing the list.
+   *
+   * @return array
+   *   Array of items to be rendered.
+   */
+  protected function getItemList(NodeInterface $node): array {
+    return [];
+  }
+
+  /**
+   * Asserts whether we can access the RSS route for a given node.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   The node being checked.
+   *
+   * @return \Drupal\Core\Access\AccessResultInterface
+   *   The access result.
+   */
+  public function checkAccess(NodeInterface $node): AccessResultInterface {
+    $bundle_entity_storage = $this->entityTypeManager->getStorage('node_type');
```

Comment:
Why you checking the bundle? Check if the node has a list page meta. Something like this:

```
$node->get('emr_entity_metas');
```

Get the meta entites, loop through, if one is list page, check it's not new, allow access.